test/src/org/apache/jorphan/util/TestJorphanUtils.java

```diff
@@ -262,6 +268,13 @@ public void testbaToHexString(){
         assertEquals("0f107f8081ff",JOrphanUtils.baToHexString(new byte[]{15,16,127,-128,-127,-1}));
     }
 
+    @Test
+    public void testbaToHexStringSeparator(){
+        assertEquals("",JOrphanUtils.baToHexString(new byte[]{}, '-'));
```

Comment:
No space after `,` in `assertEquals()`.
I see that some of the existing asserts don't have a space here...but others do.
I think the added tests should be consistent (another added test **does** have a space)

Reply:
Done